### PR TITLE
Additional query per dougbu

### PR DESCRIPTION
**Pull request by missymessa**
### To double check:

* [ ] The right tests are in and and the right validation has happened.  Guidance:  https://github.com/dotnet/arcade/tree/main/Documentation/Validation

For: https://github.com/dotnet/arcade/issues/10224

### CODE REVIEW

**Review by AlitzelMendez — DISMISSED** [review]

File: Documentation/AzureDevOps/TestReportingQueries.md [hunk 1]
```diff
@@ -139,6 +139,57 @@ subtable
...
+- `branch`: Name of the target branch
+- `testName`: Test name
+- `excludeAlwaysPassing`: Set to true to filter out tests that are always passing.
+
+```
+let ts = 30d;                                                  // Timespan value to find test results after. 
+let definition = "";                                           // Optional: The name of the build definition, leave as empty string for all build definitions.
+let repo = "dotnet/aspnetcore";                                // Optional: set to empty string if you want results from all repositories
+let branch = "";                                               // Optional: The name of the target branch the test ran against.
+let testName="Templates.Test.GrpcTemplateTest.GrpcTemplate";   // Optional: The name of the test
```
Comment (dougbu) [inline Documentation/AzureDevOps/TestReportingQueries.md:162]: nit: Shouldn't this (and maybe `repo`) be blank by default❔

File: Documentation/AzureDevOps/TestReportingQueries.md [hunk 2]
```diff
@@ -139,6 +139,57 @@ subtable
 
 :part_alternation_mark: [Link](https://dataexplorer.azure.com/clusters/engsrvprod/databases/engineeringdata?query=H4sIAAAAAAAAA51Uy24TQRC8+ytakSKthYkThQOK5UiBgLgEI5MfaHt64yGzM6t52N6IA7/B7/ElqZkljmPlgPDJ24+q6ureHY9pLtFrCcReqBG2kaKjZBfGLe+JraLWy1oQZlokbRTV3jVUszba3pFKkuufHqOEOEIdQFbSFcwQtTGkrdJLjnot5GrAb0Akihay4rV2fkSD8Tj3eKmdlxFthKygANjRM5Ss3AadUSx0ZBZqOQTI3qwQ8nszaBu9U2kp6oQ+bS/oqi9fMdJ0dnp6XDrJc5ReqbOm62HA3+vIDV7e+mTRBFKP1jxiAj7M8B0FvaWcHlFwpOOfX78DybaVZZ5qzQa2YJAdiA70/vzk/PiEBkZgTqApnb1TE3r1Bwm3upHQsu2xSpOX1qHtSLloJY7ZL1nJ0WSvadZG7SybCwpS1ihNG6EV5mA3uqbOJcrOAyskAxVll4z9ZPCgo8s2FraQFpEXRsB49YC5r2U9a8MtnAnfU9Ow7wY/s2twZI5eH69hKF1Oie9cFcOwnI6u66qXDd1Ho1JZWLoceX46zOSeHuEbdjWz+Ua7jy5B+SWdgjgUCfoBV5IawS6dnyJWHZYPR6TEuqYkP2ODJfrmlTIOszqPMG14Wz1PNKRFRx/y3V9Lra3OBn/lBqeTrej/Xfk7iLDxC4fVpJjHe5Ebbv/XwxeDQsc+0TNtmAyelrWDK0P/9co4d59aqg5EDXH4LxBR2nr3Ayf8rwPj+m/wxdhzc54drKJTLkFOtVvOkPBWPUWLuCE8P1zDdFcPNhfZzJPtE6UFAp1XeBlhxSu8k0dFwXcxzQQAAA==) to query editor
 
+## Overall Average Pass Rate for a Test (Including Retries)
+
+Query suggested by @dougbu for getting the overall average pass rate for tests, including retries.
```
Comment (dougbu) [inline Documentation/AzureDevOps/TestReportingQueries.md:144]: I feel special

File: Documentation/AzureDevOps/TestReportingQueries.md [hunk 3]
```diff
@@ -139,6 +139,57 @@ subtable
...
+- `definition`: Build definition name
+- `repo`: Repository to filter on. Set to empty string to inclue all repositories.
+- `branch`: Name of the target branch
+- `testName`: Test name
+- `excludeAlwaysPassing`: Set to true to filter out tests that are always passing.
+
+```
+let ts = 30d;                                                  // Timespan value to find test results after. 
+let definition = "";                                           // Optional: The name of the build definition, leave as empty string for all build definitions.
+let repo = "dotnet/aspnetcore";                                // Optional: set to empty string if you want results from all repositories
```
Comment (dougbu) [inline Documentation/AzureDevOps/TestReportingQueries.md:160]: Oh, all of @danmoseley's team should feel special 😀

**Review by dougbu — APPROVED** [review]

File: Documentation/AzureDevOps/TestReportingQueries.md [hunk 4]
```diff
@@ -139,6 +139,57 @@ subtable
...
 
+## Overall Average Pass Rate for a Test (Including Retries)
+
+Query suggested by @dougbu for getting the overall average pass rate for tests, including retries. 
+
+<details>
+  <summary>Expand for query</summary>
+
+Variables: 
+- `ts`: [Kusto timespan format](https://docs.microsoft.com/en-us/azure/data-explorer/kusto/query/scalar-data-types/timespan).
```
[the remark this answers is not in the document]
Comment (missymessa) [reply Documentation/AzureDevOps/TestReportingQueries.md:150]: You mean adding the default value in there?

File: Documentation/AzureDevOps/TestReportingQueries.md (hunk 1 above)
Comment (missymessa) [reply Documentation/AzureDevOps/TestReportingQueries.md:162]: Other queries have `dotnet/runtime` as a default just for demonstration purposes. I'm fine leaving `dotnet/aspnetcore` in here as a default.

File: Documentation/AzureDevOps/TestReportingQueries.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (missymessa) [reply Documentation/AzureDevOps/TestReportingQueries.md]: I saw what your meant :)  Included that bit.

**Review by AlitzelMendez — APPROVED** [review]

**Pull request merged**